### ISSUE TITLE
There is no icon for email !!

### ISSUE THREAD

**Issue by gaurav552** [atisawd/boxicons#304]
I just wanted to let you know when I searched for 'email' I got 1 icon for voice mail and when I searched for 'mail' I got 4 match for gmail, mailchimp, mail-send and voicemail.

So I would recommend adding a couple of icon for email specifically, the @ symbol, or even the snail mail letter icon, something like that.

**Comment by kerryj89** [comment]
The icon for @ is `at` and email you are probably looking for `envelope`, `paper-plane` or `send` icon.

I agree however that these icons should be tagged so that they are easier to find.  For now you just have to think in literal terms of what they are and not so much the common action associated with it.

**Comment by calvinomiguel** [comment]
Yep, the icon is not called emmail but `envelope` `<i class='bx bx-envelope'></i>` Close the issue if this is solved.

**Comment by sophiastr** [comment]
i want to deal with this issue.
I am interested in contributing to the project you are developing.
Do you still need help?
I'm a beginner but I will try really hard.
